src/components/AttachmentDragAndDrop.vue

```diff
@@ -207,8 +207,4 @@ export default {
 		margin: 20px 20px 60px 20px;
 	}
 
-	.modal__content button {
```

Comment:
Any reason to remove that? Seems unrelated.

Reply:
The "upload attachment" Button is bad formatted if this is not removed